src/main/resources/view/MainWindow.fxml

```diff
@@ -7,54 +7,48 @@
 <?import javafx.scene.control.MenuItem?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.Scene?>
 <?import javafx.stage.Stage?>
-<fx:root xmlns:fx="http://javafx.com/fxml/1" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8"
-         title="FoodRem" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
-    <icons>
-        <Image url="@/images/address_book_32.png"/>
-    </icons>
-    <scene>
-        <Scene>
-            <stylesheets>
-                <URL value="@DarkTheme.css"/>
-                <URL value="@Extensions.css"/>
-            </stylesheets>
+<fx:root xmlns:fx="http://javafx.com/fxml/1" type="javafx.stage.Stage"
+  xmlns="http://javafx.com/javafx/8" title="FoodRem" minWidth="450" minHeight="450" onCloseRequest="#handleExit">
```

Comment:
When I go fullscreen on a larger screen, it seems that the bottom "footer" is rather large so the bottom half is grey/white. Will it be possible to style it such that we can make the full screen match the background color (and the grey/white footer be fixed to a certain size) regardless of how we scale the application window size?

Reply:
That's a valid point, I noticed that as well, it has something to do with the width of the bottom footer stretching beyond the original size. As it is not crucial, I decided to leave it here for MVP, but will take note to fix it in a future PR. Thanks for the good observation!